<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="Amaze"
           tests="1"
           failures="1"
           errors="0"
           skipped="0"
           time="0.000">
  <properties />
            <testcase name="closeDriver" time="0.000" classname="base.Config">
                <failure type="java.lang.NumberFormatException"
                  message="For input string: &quot;0 megabytes&quot;"
              >
<![CDATA[java.lang.NumberFormatException: For input string: "0 megabytes"
  java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
  java.base/java.lang.Integer.parseInt(Integer.java:668)
  java.base/java.lang.Integer.parseInt(Integer.java:786)
  utils.SendMail.mailFunction(SendMail.java:124)
  base.Config.closeDriver(Config.java:91)
  java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
  java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.base/java.lang.reflect.Method.invoke(Method.java:568)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
  org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:515)
  org.testng.internal.Invoker.invokeConfigurations(Invoker.java:217)
  org.testng.internal.Invoker.invokeConfigurations(Invoker.java:144)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:357)
  org.testng.SuiteRunner.run(SuiteRunner.java:289)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
  org.testng.TestNG.runSuites(TestNG.java:1133)
  org.testng.TestNG.run(TestNG.java:1104)
  com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)
  com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)
]]>
      </failure>

          </testcase>
  
  
  
</testsuite>
